File: 1110.c
#include <stdio.h>

int main(void) {
	int n = 0;
	scanf("%d", &n);
	int num = n;
	int c = 0;
    while(1) {
		num =  num % 10 *10+ (num / 10 + num % 10)%10;
		c++;
		if (num == n) {
			break;
		}
		
	}
	printf("%d\n", c);
	return 0;
}

File: 1110.py
n = int(input())
count = 0
num = n
while True:
    num = (int(str(num)[0]) + int(str(num)[1]))%10 + (int(str(num)[1]) * 10)
    count += 1
    if num == n:
        print(count)
        break